bokeh/plotting.py

```diff
@@ -318,6 +318,19 @@ def figure(**kwargs):
        figure : a new :class:`Plot <bokeh.models.plots.Plot>`
 
     '''
+    if 'plot_width' in kwargs and 'width' in kwargs:
+        warnings.warn("figure() called but both plot_width and width supplied, supply only one")
+        return
+    if 'plot_height' in kwargs and 'height' in kwargs:
+        warnings.warn("figure() called but both plot_height and height supplied, supply only one")
+        return
```

Comment:
This should either raise an exception (instead of a warning) or, pick one the values to use (and be clear about which in the warning)